### PR TITLE
Origin/tech fair2022

### PR DESCRIPTION
**Pull request by SpartanApple**

### CODE REVIEW

**Review by modernNeo — COMMENTED** [review]
how did you generate this `csss_logo.png` ? cause even when viewing via the PR diff views, it loads a full 3-4 seconds after the `techFair_logo.png` below it....and it makes my PyCharm and native desktop picture app glitch out when trying to load it.
Screenshot from 2022-09-05 13-40-38

**Comment by SpartanApple** [comment]
@modernNeo I lowered the resolution of the CSSS logo down from 20000x20000 :). Should be good to go

File: csss-site/src/events/tech_fair/templates/tech_fair/2022/tech_fair_main.html [hunk 1]
```diff
@@ -0,0 +1,186 @@
+{% load staticfiles %}
```
Comment (modernNeo) [inline csss-site/src/events/tech_fair/templates/tech_fair/2022/tech_fair_main.html:1]: fix https://dev.sfucsss.org/PR-484/events/tech_fair/2022/main

File: csss-site/src/events/tech_fair/templates/tech_fair/2022/tech_fair_main.html [hunk 2]
```diff
@@ -0,0 +1,186 @@
...
+        frame.style.height = `${frame.contentWindow.document.body.offsetHeight}px`;
+    })
+  </script>
+</head>
+
+<body>
+
+ 
+ <div class="topnav">
+  <img class="logo" src="{% static 'csss_logo.png' %}" alt="Computing Science Student Soicety Logo">
```
Comment (modernNeo) [inline csss-site/src/events/tech_fair/templates/tech_fair/2022/tech_fair_main.html:26]: 1. you have a typo
2. These are supposed to link back to the main page

File: csss-site/src/events/tech_fair/templates/tech_fair/2022/tech_fair_main.html [hunk 3]
```diff
@@ -0,0 +1,186 @@
...
+<html lang="en">
+
+
+	<meta charset="UTF-8">
+	<link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap@4.3.1/dist/css/bootstrap.min.css" integrity="sha384-ggOyR0iXCbMQv3Xipma34MD+dH/1fQ784/j6cY/iJTQUOhcWr7x9JvoRxT2MZw1T" crossorigin="anonymous">
+	<link rel="stylesheet" type="text/css" href="{% static 'tech_fair_css.css' %}"/>
+	<title>SFU Tech Fair 2022</title>
+
+
+  <script>
```
Comment (modernNeo) [inline csss-site/src/events/tech_fair/templates/tech_fair/2022/tech_fair_main.html:13]: this problem is back 
Screenshot from 2022-09-05 14-55-49
Screenshot from 2022-09-05 14-55-58

**Review by modernNeo — CHANGES_REQUESTED** [review]
some issues back from the dead

File: csss-site/src/events/tech_fair/templates/tech_fair/2022/tech_fair_main.html [hunk 4]
```diff
@@ -0,0 +1,177 @@
...
+	<link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap@4.3.1/dist/css/bootstrap.min.css" integrity="sha384-ggOyR0iXCbMQv3Xipma34MD+dH/1fQ784/j6cY/iJTQUOhcWr7x9JvoRxT2MZw1T" crossorigin="anonymous">
+	<link rel="stylesheet" type="text/css" href="{% static 'tech_fair_css.css' %}"/>
+	<title>SFU Tech Fair 2022</title>
+</head>
+
+<body>
+
+ 
+ <div class="topnav">
+  <a href="{{ URL_ROOT }}"> <img class="logo" src="{% static 'csss_logo.png' %}" alt="Computing Science Student Soicety Logo"></a>
```
Comment (modernNeo) [inline csss-site/src/events/tech_fair/templates/tech_fair/2022/tech_fair_main.html:17]: typo in alt

File: csss-site/src/events/tech_fair/templates/tech_fair/2022/tech_fair_main.html [hunk 5]
```diff
@@ -0,0 +1,177 @@
+{% load staticfiles %}
+
+<!DOCTYPE html>
+<html lang="en">
+
+
+	<meta charset="UTF-8">
+	<link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap@4.3.1/dist/css/bootstrap.min.css" integrity="sha384-ggOyR0iXCbMQv3Xipma34MD+dH/1fQ784/j6cY/iJTQUOhcWr7x9JvoRxT2MZw1T" crossorigin="anonymous">
+	<link rel="stylesheet" type="text/css" href="{% static 'tech_fair_css.css' %}"/>
```
Comment (modernNeo) [inline csss-site/src/events/tech_fair/templates/tech_fair/2022/tech_fair_main.html:9]: please match existing convention w/ regards to naming the `.css` files

**Review by modernNeo — CHANGES_REQUESTED** [review]

File: csss-site/src/events/tech_fair/templates/tech_fair/2022/tech_fair_main.html (hunk 4 above)
Comment (SpartanApple) [reply csss-site/src/events/tech_fair/templates/tech_fair/2022/tech_fair_main.html]: Fixed

File: csss-site/src/events/tech_fair/templates/tech_fair/2022/tech_fair_main.html (hunk 5 above)
Comment (SpartanApple) [reply csss-site/src/events/tech_fair/templates/tech_fair/2022/tech_fair_main.html]: Fixed. Resolved.

**Pull request merged**